expression/builtin_miscellaneous.go

```diff
@@ -534,17 +533,23 @@ type isIPv4CompatFunctionClass struct {
 }
 
 func (c *isIPv4CompatFunctionClass) getFunction(args []Expression, ctx context.Context) (builtinFunc, error) {
-	sig := &builtinIsIPv4CompatSig{newBaseBuiltinFunc(args, ctx)}
+	sig := &builtinIsIPv4PrefixedSig{newBaseBuiltinFunc(args, ctx), false}
 	return sig.setSelf(sig), errors.Trace(c.verifyArgs(args))
 }
 
-type builtinIsIPv4CompatSig struct {
+type builtinIsIPv4PrefixedSig struct {
 	baseBuiltinFunc
+	isOneInSixthBytePair bool
 }
 
-// eval evals a builtinIsIPv4CompatSig.
+// eval evals a builtinIsIPv4MappedSig and builtinIsIPv4CompatSig.
```

Comment:
there are no builtinIsIPv4MappedSig and builtinIsIPv4CompatSig,
`evals Is_IPv4_Mapped and Is_IPv4_Compat` may be ok.